app/src/main/java/uk/co/ashtonbrsc/intentexplode/Explode.java

```diff
@@ -219,15 +219,15 @@ private void rememberIntent(Intent original) {
 
         if (originalExtras != null) {
             // bugfix #14: collect extras that are lost in the intent <-> string conversion
-            Bundle additionalExtas = new Bundle(originalExtras);
+            Bundle additionalExtasVar = new Bundle(originalExtras);
```

Comment:
I agree with the reason for this change but "additionalExtasVar" doesn't seem like a well named variable.

Also there's a spelling mistake in the name which we can also fix at the same time: Extas -> Extras.